test/src/test/java/org/corfudb/runtime/checkpoint/CheckpointLivenessUpdaterUnitTest.java

```diff
@@ -34,10 +37,13 @@ public class CheckpointLivenessUpdaterUnitTest {
             (CorfuStoreEntry<? extends Message, ? extends Message, ? extends Message>) mock(CorfuStoreEntry.class);
 
     private static final Duration INTERVAL = Duration.ofSeconds(15);
+    private static final int LONG_INTERVAL = 60;
+    private static final int NUM_REPETITIONS = 5;
     private static final TableName tableName = TableName.newBuilder().setNamespace("TestNamespace").setTableName("TestTableName").build();
 
     private CheckpointLivenessUpdater livenessUpdater;
 
+    @BeforeEach
```

Comment:
AFAIK, @BeforeEach is essentially equivalent to @Before: https://stackoverflow.com/questions/20295578/difference-between-before-beforeclass-beforeeach-and-beforeall

Reply:
It also looks like there is a race in the setup function here, where the mocked behaviour can take effect after updateHeartbeat executes. I think this can be mitigated by initializing the CheckpointLivenessUpdater after defining the mocked behaviour.

Reply:
Without @beforeeach, @RepeatedTest fails. Hence I am using that annotation as well.

Reply:
Got it - It can replace the usage of @before.

Reply:
Removed usage of @BeforeEach.